From df7a2df15578b93995f7f7fd0c615ad18b78eda4 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Fri, 12 Jun 2026 11:22:24 +0100
Subject: [PATCH] debug-builds: set persist.sys.phh.debug_build=true

this property gates all securize behaviour in rw-system.sh,
phh-on-boot.sh, and phh-prop-handler.sh. when set to true,
securize fingerprint spoofing, build tag hiding, and vbmeta
state spoofing are all skipped, keeping the device in a
debuggable state for diagnostics.
---
 base.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base.mk b/base.mk
index 2909d5f..e1aa0cd 100644
--- a/base.mk
+++ b/base.mk
@@ -40,6 +40,7 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 
 # Huawei HiSuite (also other OEM custom programs I guess) it's of no use in AOSP builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
+    persist.sys.phh.debug_build=true
     ro.cust.cdrom=/dev/null
 
 # VNDK config files
-- 
2.54.0

